.release-please-manifest.json

```diff
@@ -0,0 +1,3 @@
+{
+    ".": "1.4.0"
```

Comment:
Any reason why this is below of the version in `_version.py`?

Reply:
I think he started from an old branch. Just solved that.